src/components/policy/src/policy/policy_table/table_struct/types.cc

```diff
@@ -376,7 +378,10 @@ bool ModuleConfig::is_valid() const {
   if (!vehicle_year.is_valid()) {
     return false;
   }
-  if (!remote_control.is_valid()) {
+  if (!user_consent_passengerRC.is_valid()) {
+    return false;
+  }
+  if (!country_consent_passengersRC.is_valid()) {
```

Comment:
As per Validation:
3. R-SDL PoliciesManager must treat the Preloaded PT as valid independently on whether "country_consent_passengersRC" field exists or omitted in Preloaded PT (that is, this field is optional for Pre-PT).
4. R-SDL PoliciesManager must treat the Updated PT as valid independently on whether "country_consent_passengersRC" field exists or omitted in Updated PT (that is, this field is optional for Updated PT).
this field should not be treated as invalid if it's absent; should this validation be present?